lib/chef/resource/openssl_x509_certificate.rb

```diff
@@ -24,7 +26,7 @@ class OpensslX509Certificate < Chef::Resource
 
       preview_resource true
       resource_name :openssl_x509_certificate
-      provides(:openssl_x509) { true } # legacy cookbook name. Cookbook will win. @todo Make this true in Chef 15
+      provides(:openssl_x509) { false } # legacy cookbook name. Cookbook will win. @todo Make this true in Chef 15
```

Comment:
that just globally disables this provides line entirely, probably not what you want.

Reply:
think you want to be using noah's api here.

Reply:
It's really just a reminder to get this fixed in Chef 15